docs/developing-contracts/messaging.md

```diff
@@ -55,24 +58,22 @@ pub fn load_bytes() -> Vec<u8> {
     result
 }
 ```
-Load bytes
+
+Reply to a message and try to decode to the specified type with `SCALE codec`. Returns `MessageId`:
```

Comment:
This one encodes the argument and sends it. No decode functions

Reply:
done